options/cf_options.cc

```diff
@@ -546,9 +546,6 @@ static std::unordered_map<std::string, OptionTypeInfo>
         {"bloom_locality",
          {offset_of(&ImmutableCFOptions::bloom_locality), OptionType::kUInt32T,
           OptionVerificationType::kNormal, OptionTypeFlags::kNone}},
-        {"rate_limit_delay_max_milliseconds",
-         {0, OptionType::kUInt, OptionVerificationType::kDeprecated,
-          OptionTypeFlags::kNone}},
```

Comment:
I guess we need to leave this. Apparently `kDeprecated` in this context means unsupported or removed.

Reply:
Good catch - same for my other PR too.